build/build.yml

```diff
@@ -22,12 +22,13 @@ steps:
 - task: NuGetCommand@2
   displayName: Restore nuget packages
 
-- powershell: |
-    Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force
+- pwsh: |
+    Install-PackageProvider -Name NuGet -Force
+# install platyPS to gernate help docs from markup files
```

Comment:
The pipeline is failing to parse this:

```
/build/build.yml: (Line: 28, Col: 5, Idx: 828) - (Line: 28, Col: 82, Idx: 905): While parsing a block mapping, did not find expected key.
```

Spacing is important in YAML, so this needs to shift over in line with the other commands. This should fix it.